src/settings/Settings.cpp

```diff
@@ -847,6 +876,13 @@ void Settings::renamings(RenameType renameType, bool& files, bool& folders, bool
     seasonDirectories =
         settings()->value(QString("RenamePattern/%1/UseSeasonDirectories").arg(renameTypeStr), true).toBool();
 }
+void Settings::renamings(RenameType renameType, bool& files, bool& folders, bool& seasonDirectories, bool& replaceDelimiter)
```

Comment:
There is no need to add another method. Adapting the existing one is fine. :)
The code isn't great to begin with, so adding another argument is ok here.